### ISSUE TITLE
Deploy button in CloudFlare does not work

### ISSUE THREAD

**Issue by Karen-Aijo** [eidam/cf-workers-status-page#79]
I click on the button and it doesn't appear as it should. It sends me to try other workers

![image](https://user-images.githubusercontent.com/80742646/152788324-c101b346-66c6-4bf5-b81a-6788738227bb.png)
![msedge_A2mtUwOpgl](https://user-images.githubusercontent.com/80742646/152788505-ab096376-83cc-4102-98ae-2b2d480a09ce.png)
![msedge_lhJtgCDOND](https://user-images.githubusercontent.com/80742646/152788526-dc7e628a-0a57-43b8-8d6d-7e56c4e6a825.png)

**Comment by supunsathsara** [comment]
After Authorize Workers (second screenshot), 
try clicking the deploy button again. It should take you to next step.

**Comment by H01001000** [comment]
Nope, I got the same issue, where even i press the second time it go back to step one (start over).

**Comment by andyrb412** [comment]
Same issue, can't find any way to progress.

**Comment by supunsathsara** [comment]
@andyrb412  @H01001000 
After Authorize GitHub with Workers step, try adding the `?authed=true` parameter to the url..
In this case `https://deploy.workers.cloudflare.com/?url=https://github.com/eidam/cf-workers-status-page?authed=true`
This should work...

**Comment by andyrb412** [comment]
Nope, just going around in circles.

Don't care anymore, I've given up.

**Comment by supunsathsara** [comment]
First check whether cloudflare got authorized correctly.
- Go to [github.com/settings/applications](https://github.com/settings/applications) and see if there is an app called `Deploy to Cloudflare Workers`. 

After got authorized correctly,
If above commented method doesn't work, try manually forking the repo and use your forked repos url for the url paramater and add the  `?authed=true` in the end..
- so it should be `https://deploy.workers.cloudflare.com/?url=<YOUR GITHUB REPO LINK>?authed=true`
